src/Http/Routing/src/Resources.resx

```diff
@@ -249,4 +249,7 @@
   <data name="RouteEndpointDataSource_RequestDelegateCannotBeCalledBeforeBuild" xml:space="preserve">
     <value>This RequestDelegate cannot be called before the final endpoint is built.</value>
   </data>
+  <data name="RegexRouteContraint_NotConfigured" xml:space="preserve">
+    <value>The route parameter '{0}' uses a regular expression constraint. If this application was configured using CreateSlimBuilder(...) or AddRoutingCore(...) it may not include the RegexInlineRouteConstraint. To register this route route constraint call SetParameterPolicy&lt;RegexInlineRouteConstraint("regex") on RouteOptions via the Configure&lt;TOptions&gt;(Action&lt;RouteOptions&gt; configureOptions) extension method.</value>
```

Comment:
```suggestion
    <value>The route parameter '{0}' uses a regex constraint. If this application was configured using CreateSlimBuilder(...) or AddRoutingCore(...) then this constraint is not registered by default. To register the regex constraint, configure route options at app startup: services.Configure&lt;RouteOptions&gt;(options => options.SetParameterPolicy&lt;RegexInlineRouteConstraint&gt;("regex"));</value>
```
I think this is more understandable. Devs don't know about `RegexInlineRouteConstraint`, they just know the alias 'regex'.

Also, this error allows them to copy and paste the code to fix their app directly from the exception message.